js-intro.js

```diff
@@ -4,58 +4,195 @@
+})
+
+
+//Logging the multiply. 
+console.log(multiplyByArr)
 
+*/
+//--------------------------
 
+// // --------------------4) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-// --------------------4) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
+/*
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+//Creation of new function to remove vowels
+const nayVow = defNoVows => {
+  let newArray1 = defNoVows.split("")
+  let newNewArray = newArray1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
+  let totesNewArray = newNewArray.join("")
+  return totesNewArray
+}
 
+//Logging the strings without vowels
+console.log(nayVow(stringWithVowels1));
+console.log(nayVow(stringWithVowels2));
 
 
-// --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
-// Copy and paste or rewrite the function. Use the test variables provided below.
+*/
+
+//--------------------------
+
+// // --------------------5) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
+// // Copy and paste or rewrite the function. Use the test variables provided below.
+
+/*
 
 var notAString1 = true
 var notAString2 = 42
 
+//Creation of new function to remove vowels
+const nayVow = defNoVows => {
+if(typeof(defNoVows)==="string") {
+    let newArray1 = defNoVows.split("")
+  let newNewArray = newArray1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
+  let totesNewArray = newNewArray.join("")
+  return totesNewArray
+} else { return "Not a string, bro"}
+}
+
+//Logging the strings without vowels (or spits out the "Not a string, bro")
+console.log(nayVow(notAString1));
+console.log(nayVow(notAString2));
+
+
+*/
+//-----------------------------
+
+
+// // --------------------6) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
+// // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
+
+/*
+
+var toonimals = [ {name: "Itchy", animal: "mouse"}, 
+{name: "Stimpy", animal: "cat"}, 
+{name: "Daffy", animal: "duck"}, 
+{name: "Scratchy", animal: "cat"}, 
+{name: "Ren", animal: "dog"}, 
+{name: "Felix", animal: "cat"}]
+
+//Creation of function that will filter out only cats from the above array
+
+const onlyCats = (array) => {
+    let newArr = array.filter(value=> value.animal==="cat")
+  return newArr
```

Comment:
Same as earlier, you can save yourself some time and just return the filtered output :)